fiftyone/core/utils.py

```diff
@@ -1069,7 +1069,6 @@ def progress_dt(pb):
 
 
 class Batcher(abc.ABC):
```

Comment:
I don't like what I did with the class naming. How about this becomes `BaseBatcher` and `BaseBatcher` becomes something like `BaseChunkyBatcher`. Meaning the batch size is determined per chunk via `_compute_batch_size()`
Would have to explain this in docstring of course 🙃